tests/components/Tabs-test.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { render, screen, fireEvent, waitFor } from '@testing-library/react';
+import Tabs from '../../components/tabs/Tabs';
+
+
+
+const defaultProps = {
+    types: [
+        {
+          internal: 'name',
+          label: 'Names'
+        },
+        {
+          internal: 'dates',
+          label: 'Dates',
+          count: 23,
+          tooltip: (<div>:wave:</div>)
+        }
+    ],
+    active: 'name',
+    switchTab: () => {}
+};
+
+const switchTab = (tab) => {
+  defaultProps.active = tab
+};
+
+describe('SearchBar', () => {
```

Comment:
searchbar?

Reply:
assuming this should say tabs. Also, we dont really need to have nested describe statements. When we execute the test suite jest prints out the test file thats being logged so we dont really need to do this IMO.

Reply:
agreed glad to see you changed your mind on this